charts/nri-metadata-injection/README.md

```diff
@@ -1,64 +1,75 @@
-| `nodeSelector`                  | Node label to use for scheduling                                                                                                                                                                                                                                                                                                          | `{}`                                |
-| `timeoutSeconds`                | Seconds to wait for a webhook to respond. The timeout value must be between 1 and 30 seconds                                                                                                                                                                                                                                              | `10`                                |
-| `tolerations`                   | List of node taints to tolerate (requires Kubernetes >= 1.6)                                                                                                                                                                                                                                                                              | `[]`                                |
-| `affinity`                      | Node affinity to use for scheduling                                                                                                                                                                                                                                                                                                       | `{}`                                |
-| `hostNetwork`                      | Enable hostNetwork                                                                                                                                                                                                                                                                                                      | `false`                                |
-| `dnsConfig`                      | Set DNS settings                                                                                                                                                                                                                                                                                                      | `{}`                                |
+# Helm installation
 
-## Example
+You can install this chart using [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) located in the
+[helm-charts repository](https://github.com/newrelic/helm-charts) or directly from this repository by adding this Helm repository:
 
-Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
+```shell
+helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
```

Comment:
copy-paste issue

Reply:
Solved in [#4a082e6](https://github.com/newrelic/k8s-metadata-injection/pull/171/commits/4a082e6389a9968fb77a01508b68f2da71551ca0)